.github/workflows/main.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
+on: [push]
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-latest
```

Comment:
Its interesting that GitHub-hosted runner only has ubuntu in linux [github hosted runner](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on) but thats okay I think because we used debian with travis as well

Reply:
In the future, we can change this to use `podman` and run our tests in a container of the kind we need.